src/Operations.jl

```diff
@@ -1051,6 +1051,12 @@ function update_package_add(ctx::Context, pkg::PackageSpec, entry::PackageEntry,
     return pkg
 end
 
+# Update registries AND read them back in.
+function update_registries(ctx::Context, registries...; kwargs...)
```

Comment:
Maybe just skip the `registries` argument? It doesn't really match up with the `copy!`.